content/6_Advanced/Multiplicative.mdx

```diff
@@ -1,24 +1,227 @@
 ---
 id: multiplicative
 title: 'Prefix Sums of Multiplicative Functions'
-author: Benjamin Qi
-description: '?'
+author: Tianqin Meng,  Benjamin Qi
+description: This module delves into the topic of prefix sum with multiplicative functions.
```

Comment:
uh perhaps a more informative description?

Reply:
okay i made it more informative plz check when ur free